builder.md

```diff
@@ -1,57 +1,180 @@
-# Builder Guidelines
-**Version 0.10.0**
+# Base16 builder specification
+**Version 0.11.0**
 
-A base16 builder is an application that can build syntax highlighting definition files for text editors by using base16 scheme files which contain a collection of colours and base16 template files which contain syntax highlighting rules.
+*The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).*
 
-Builders are designed for theme maintainers' ease of use. Theme maintainers should provide built versions of their theme so the end user doesn't need to be aware of the builder.
+## 1. Introduction
 
-## File Structure
+"Building" refers to replacing placeholders on base16 _[template
+files](./file.md)_ with colors from base16 _[schemes](./styling.md)_.
 
-### Schemes Repository
+A template is a mustache file that acts as a "blueprint": it represents how to
+output the scheme into other file formats, or (more commonly) how to output a
```

Comment:
I would leave off the whole "or" clause here because it's already included in the first part of the sentence.